src/node_quic_util.h

```diff
@@ -435,16 +431,12 @@ class Timer {
   static void Free(Timer* timer);
 
  private:
-  inline void OnTimeout() {
-    fn_(data_);
-  }
-
   static void OnTimeout(uv_timer_t* timer);
   static void CleanupHook(void* data);
 
   bool stopped_;
   Environment* env_;
-  std::function<void(void* data)> fn_;
+  std::function<void()> fn_;
   uv_timer_t timer_;
   void* data_;
```

Comment:
Looks like the `data_` member is not used anymore:
```console
In file included from ../src/node_quic_util.cc:1:
../src/node_quic_util.h:440:9: warning: private field 'data_' is not used [-Wunused-private-field]
  void* data_;
        ^
1 warning generated.
```

Reply:
Yup, done :+1: